prow/plugins/override/override.go

```diff
@@ -37,7 +37,7 @@ import (
 const pluginName = "override"
 
 var (
-	overrideRe = regexp.MustCompile(`(?mi)^/override (.+?)\s*$`)
+	overrideRe = regexp.MustCompile(`(?mi)^/override( (.+?)\s*)?$`)
```

Comment:
This doesn't need to be a capturing group. Adding an additional capturing group to check if the value is empty doesn't provide any more value than using the existing capture group and making this new group a non-capturing group.